snap-template/snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: my-snap  # the name of the snap
+version: 0  # the version of the snap
+summary: This is my-snap's summary  # 79 char long summary
+description: |
+  This is my-snap's description  # a longer description for the snap
+  which can expand multiple lines.
+confinement: devmode  # start with devmode, then use "strict" to enforce system
+                      # access only via declared interfaces
+
+apps:  # declarative apps come in second place
+  my-snap:  # if only one command is exposed, please use
+            # [app name == snap name] to have a shorten command executable.
+    command: my-command
+
+parts:
+  my-part:  # Replace with a part name of your liking
+    # Get more information about plugins by running
```

Comment:
If it's going to be on different lines from the command, I would suggest adding a colon at the end of this line